sorting/radix_sort/radix_sort.cpp

```diff
@@ -0,0 +1,130 @@
+/*
+    Radix Sort Algorithm
+    --------------------
+
+    Time Complexity - O(d*(n + b)),
+        where d is the max number of digits, n is the size of the input & b is the base of the system.
+
+    Space Complexity - O(n + b),
+        where n is the size of the input & b is the base of the system.
+*/
+
+#include <iostream>
+#include <vector>
+#include <cstdlib>
+#include <ctime>
+
+using namespace std;
+
+// Gets the maximum element in array (To find the max number of digits).
+int getMax(vector<int>& arr) {
+    if (arr.empty()) { 
+        cerr << "The Array to Sort is Empty!" << endl; 
+        exit(1);
+    }
+    int max = arr[0];
+    for (int num : arr) {
+        if (num > max) {
+            max = num;
+        }
+    }
+    return max;
+}
```

Comment:
Now that you're using `vector`, there's an inbuilt method `*max_element`. Use that instead of writing your own!